src/randoop/main/GenInputsAbstract.java

```diff
@@ -219,20 +184,18 @@ public GenInputsAbstract(String command, String pitch,
   @OptionGroup("Values used in tests")
   @Option("Use null as an input with the given frequency")
   public static double null_ratio = 0;
-  
+
   /**
-   * Never use null as input to methods or constructors.
+   * Suppress use of null when no other argument value can be generated.
    * 
-   * This option causes Randoop to abandon the method call rather than providing
-   * null as an input, when no non-null value of the appropriate type is available.
+   * If true, Randoop will fail input generation rather than use null as an 
+   * input.
    * 
-   * To ask Randoop to calls methods with null with greater frequency,
-   * see option --null-ratio.
+   * Does not affect the behavior based on {@link #null_ratio}.
```

Comment:
Yes, please.  The rationale is that more people will read it, and that even though this is marginally less nice for the HTML API documentation, it's still better than having two copies of the documentation.
A feature for the Options doclet that creates the option listing in the manual would be to automatically change occurrences of `{@link #null_ratio}` to `--null-ratio`, but that doesn't seem like a high priority right now.